arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -1247,115 +1165,81 @@
   "definitions": {
     "CheckAvailabilityParameters": {
       "properties": {
-        "id": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource Id"
-        },
-        "name": {
-          "type": "string",
-          "description": "Resource name"
-        },
-        "type": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource type"
-        },
-        "location": {
-          "type": "string",
-          "description": "Resource location"
-        },
-        "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "Resource tags"
-        },
-        "sku": {
-          "$ref": "#/definitions/Sku",
-          "description": "The sku of the created namespace"
-        },
         "isAvailiable": {
           "type": "boolean",
-          "description": "True if the name is available and can be used to create new Namespace/NotificationHub. Otherwise false."
+          "description": "True if the name is available and can be used to create new namespace/notification hub, otherwise false"
         }
       },
       "required": [
-        "name",
-        "location"
+        "name"
+      ],
+      "allOf": [
```

Comment:
Thanks for catching that. I did not go through this document to check for errors but simply updated to fulfill requirements around capitalization and description. Will delete that.